### ISSUE TITLE
Add Support for Docker secrets

### ISSUE THREAD

**Issue by satyadeep** [nosqlclient/nosqlclient#490]
Add support for docker secrets so that we can use the secrets file to provide connection URLs with passwords in them and other sensitive information instead of typing them in clear text in the docker-compose file.

**Comment by rsercano** [comment]
Hello @satyadeep do you mean nosqlclient's internal mongodb connection ?

**Comment by satyadeep** [comment]
Hi @rsercano I meant when defining connections in the docker-compose file

Instead of defining them like this,

```
 environment:
            - 'MONGO_URL=mongodb://mongodbroot:MehTJLV9dSQt88pVuYIkKqNq2+k=@mongodb:27017"'
            - 'MONGOCLIENT_AUTH=true'
            - 'MONGOCLIENT_USERNAME=admin'
            - 'MONGOCLIENT_PASSWORD=mongopassword'

```
Use Docker secrets files, like below
```
environment:
            - 'MONGO_URL_FILE=/run/secrets/mongoconnection'
            - 'MONGOCLIENT_DEFAULT_CONNECTION_URL_FILE=/run/secrets/mongoconnection'
            - 'MONGOCLIENT_AUTH=true'
            - 'MONGOCLIENT_USERNAME=admin'
            - 'MONGOCLIENT_PASSWORD_FILE=/run/secrets/mongoclient'
```
Opened this issue to create a pull request for your review.

Thanks!

**Comment by rsercano** [comment]
@satyadeep Oh thank you so much, actually a PR would save a lot of time of mine.

**Comment by rsercano** [comment]
fixed with #491
